com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -123,8 +152,29 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
             m_ForwardLights = new ForwardLights();
             //m_DeferredLights.LightCulling = data.lightCulling;
             this.m_RenderingMode = data.renderingMode;
+            this.m_DepthPrepassMode = data.depthPrepassMode;
+            this.m_DepthPrimingMode = data.depthPrimingMode;
             this.usesRenderPass = data.useNativeRenderPass;
 
+#if !UNITY_ANDROID && !UNITY_IOS
+            this.m_CanUseDepthPriming = true;
+
+#if UNITY_EDITOR_WIN || UNITY_STANDALONE_WIN
+            if (CultureInfo.InvariantCulture.CompareInfo.IndexOf(SystemInfo.graphicsDeviceVendor, "ARM", CompareOptions.IgnoreCase) >= 0)
+            {
+                this.m_CanUseDepthPriming = false;
+            }
```

Comment:
TODO: check if https://docs.unity3d.com/ScriptReference/SystemInfo-hasHiddenSurfaceRemovalOnGPU.html is enabled on Mali. If it's not at least on CPU my understanding is that we could improve it.